src/cclib/parser/gaussianparser.py

```diff
@@ -223,6 +223,109 @@ def extract(self, inputfile, line):
             natom = int(line.split()[1])
             self.set_attribute('natom', natom)
 
+        # Dipole moment
+        # e.g.
+        #  Dipole moment (field-independent basis, Debye):
+        #    X=              0.0000    Y=              0.0000    Z=              0.0930
+        if line[1:39] == "Dipole moment (field-independent basis":
+
+            self.updateprogress(inputfile, "Dipole and Higher Moments", self.fupdate)
+
+            self.reference = [0.0, 0.0, 0.0]
+            self.moments = [self.reference]
+
+            tokens = inputfile.next().split()
```

Comment:
Oh, I'm referring to explicit slicing (`dipole = float(line[a:b]), float(line[b:c]), float(line[c:d])`) rather than splitting.